sdk/src/main/java/io/horizen/cryptolibprovider/CommonCircuit.java

```diff
@@ -71,8 +71,22 @@ public static WithdrawalCertificate createWithdrawalCertificate(WithdrawalEpochC
         );
     }
 
+    public static WithdrawalCertificate createWithdrawalCertificateWithBtrFreeAndFtMinAmountSwapped(WithdrawalEpochCertificate cert, Enumeration.Value sidechainCreationVersion) {
```

Comment:
Actually it's more general purpose than only sc2sc; the `createWithdrawalCertificate` should not be called every time, but only when we need to swap the `btrFee` and `ftMinAmount` (see [here](https://github.com/HorizenOfficial/Sidechains-SDK/blob/dev/sdk/src/main/scala/io/horizen/block/SidechainCommitmentTree.scala#L74)) otherwise we double swap them ending up with the two parameters in the initial order

Reply:
Ok, it might be called in future by other modules, therefore it make sense to leave it there